gcloud/bigtable/table.py

```diff
@@ -139,6 +139,36 @@ def create(self, initial_split_keys=None):
         # We expect a `._generated.bigtable_table_data_pb2.Table`
         client._table_stub.CreateTable(request_pb, client.timeout_seconds)
 
+    def rename(self, new_table_id):
+        """Rename this table.
+
+        .. note::
+
+            This cannot be used to move tables between clusters,
+            zones, or projects.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+                ``BigtableTableService.RenameTable is not yet implemented``
+
+            when this method is used. It's unclear when this method will
+            actually be supported by the API.
```

Comment:
Just trying to match the [service definition][1] (see [`RenameTableRequest`][2] def'n as well).

I actually haven't checked if the method works in awhile. Doing it now and will post back what I found out.

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/5d84776473347513db0b6c44e4f8503837a0d9fa/bigtable-protos/src/main/proto/google/bigtable/admin/table/v1/bigtable_table_service.proto#L56-L58
[2]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/5d84776473347513db0b6c44e4f8503837a0d9fa/bigtable-protos/src/main/proto/google/bigtable/admin/table/v1/bigtable_table_service_messages.proto#L77-L84